packages/service-library/src/web-api/api-contracts/privacy-scan-types/privacy-scan-run-request.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { PrivacyScanConfiguration, StorageDocument } from 'storage-documents';
+import { ReportGroup, Website } from '../scan-run-request';
+
+export type PrivacyScanConfigDefinition = Omit<PrivacyScanConfiguration, keyof StorageDocument>;
+
+export interface PrivacyScanRequest {
```

Comment:
no need for this type, we will use existing structure with `privacyScan?: boolean;` flag